GitCommands/Settings/AppSettings.cs

```diff
@@ -1179,13 +1179,11 @@ public static void SaveSettings()
         public static void LoadSettings()
         {
             Action<Encoding> addEncoding = delegate(Encoding e) { AvailableEncodings[e.HeaderName] = e; };
-            addEncoding(Encoding.Default);
-            addEncoding(new ASCIIEncoding());
-            addEncoding(new UnicodeEncoding());
-            addEncoding(new UTF7Encoding());
-            addEncoding(new UTF8Encoding(false));
-            addEncoding(System.Text.Encoding.GetEncoding("CP852"));
-
+            foreach(var enc in Encoding.GetEncodings())
+            {
+                addEncoding(enc.GetEncoding());
+            }
```

Comment:
Good day! We needed cp866, but add a single character encoding seemed to me unfortunate decision. I'll think about what can be done in a convenient way.